### PR TITLE
Added Mesos to CNI users.

### PR DESCRIPTION
**Pull request by jieyu**
Mesos recently added support for CNI. The documentation can be found here:
https://github.com/apache/mesos/blob/master/docs/cni.md

### CODE REVIEW

**Comment by steveeJ** [comment]
Please adjust the commit message(s) to conform to https://github.com/containernetworking/cni/blob/master/CONTRIBUTING.md#format-of-the-commit-message, thanks a lot!

**Comment by jieyu** [comment]
@steveeJ Thanks for the pointer! Fixed it.

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ To avoid duplication, we think it is prudent to define a common interface betwee
 - [Weave - a multi-host Docker network](https://github.com/weaveworks/weave)
 - [Project Calico - a layer 3 virtual network](https://github.com/projectcalico/calico-cni)
 - [Contiv Networking - policy networking for various use cases](https://github.com/contiv/netplugin)
+- [Mesos - a distribute system kernel](https://github.com/apache/mesos/blob/master/docs/cni.md)
```
Comment (jonboulle) [inline README.md]: nit: distribute system -> distributed systems

**Comment by jonboulle** [comment]
woohoo, excited to see this!

**Comment by jieyu** [comment]
@jonboulle fixed the typo. thanks for spotting it.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ To avoid duplication, we think it is prudent to define a common interface betwee
 - [Weave - a multi-host Docker network](https://github.com/weaveworks/weave)
 - [Project Calico - a layer 3 virtual network](https://github.com/projectcalico/calico-cni)
 - [Contiv Networking - policy networking for various use cases](https://github.com/contiv/netplugin)
+- [Mesos - a distributed system kernel](https://github.com/apache/mesos/blob/master/docs/cni.md)
```
Comment (jonboulle) [inline README.md]: sorry, but you missed system -> systems :-)

Comment (jonboulle) [inline README.md]: (from http://mesos.apache.org/ )

**Comment by jieyu** [comment]
Looks like the test is flaky:)

**Comment by jieyu** [comment]
Any more comments? I think the CI failure is not due to my change obviously :)

**Comment by jieyu** [comment]
@steveeJ @jonboulle any more suggestions? Can you merge this PR? Thanks!

**Comment by squaremo** [comment]
Yep, the CI is flaking. Looks good now -- thanks @jieyu, and thanks @jonboulle for checking it through.

**Pull request merged**